File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        for (int i=0;i<nums.size()-1;++i){
            if (nums[i]==nums[i+1]){
                nums[i] = 101;
            }
        }
        int l = 0 ; 
        int i = 0 ; 
        while(i<nums.size()){
            if (nums[i]!=101){
                swap(nums[l],nums[i]);
                l++;
            }
            i++;
        }
        return l;
    }
};

File: Data Structures & Algorithms/jump-game/submission-0.py
class Solution:
    def canJump(self, nums: List[int]) -> bool:
        i = len(nums)-1
        j = len(nums)-2
        while(j>=0):
            if (nums[j]>=i-j):
                i = j
            j -=1   
        return i==0        
        